core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/scatterplot/ScatterplotOpDesc.java

```diff
@@ -0,0 +1,68 @@
+package edu.uci.ics.texera.workflow.operators.visualization.scatterplot;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaTitle;
+import edu.uci.ics.texera.workflow.common.metadata.InputPort;
+import edu.uci.ics.texera.workflow.common.metadata.OperatorGroupConstants;
+import edu.uci.ics.texera.workflow.common.metadata.OperatorInfo;
+import edu.uci.ics.texera.workflow.common.metadata.OutputPort;
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName;
+import edu.uci.ics.texera.workflow.common.operators.OneToOneOpExecConfig;
+import edu.uci.ics.texera.workflow.common.tuple.schema.Attribute;
+import edu.uci.ics.texera.workflow.common.tuple.schema.AttributeType;
+import edu.uci.ics.texera.workflow.common.tuple.schema.Schema;
+import edu.uci.ics.texera.workflow.operators.visualization.VisualizationConstants;
+import edu.uci.ics.texera.workflow.operators.visualization.VisualizationOperator;
+import static java.util.Collections.singletonList;
+import static scala.collection.JavaConverters.asScalaBuffer;
+
+/**
+ * Scatterplot operator to visualize the result as a scatterplot
+ * This is the description of the operator
+ */
+
+public class ScatterplotOpDesc extends VisualizationOperator {
+    @JsonProperty(value = "X-Column", required = true)
+    @AutofillAttributeName
+    public String xColumn;
+
+    @JsonProperty(value = "Y-Column", required = true)
+    @AutofillAttributeName
+    public String yColumn;
+
+    @JsonProperty(defaultValue = "false")
+    @JsonSchemaTitle("Geometric")
+    @JsonPropertyDescription("plot on a map")
+    public boolean isGeometric;
+
+    @Override
+    public String chartType() {
+        if(isGeometric)
```

Comment:
please add curly braces `{ }`, we typically avoid single-line if without curly braces in Java code.